packages/ses/test/frozen-primordials.test.js

```diff
@@ -0,0 +1,19 @@
+import test from 'tape';
+import '../ses.js';
```

Comment:
I recommend changing this to `'../lockdown.js'` since this test does not depend on the module layer. This will help us tease the layers into separate packages later, should we choose to.

Reply:
Amusingly, I was able to do this only after rebasing on #472 .